src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -3,7 +3,18 @@
 public class ConcurConstants {
     public static final String AUTHORIZATION_PROPERTY = "Authorization";
     public static final String BASIC_AUTHENTICATION_SCHEME = "Basic";
-    
+    public static final String OAUTH_AUTHENTICATION_SCHEME = "OAuth ";
```

Comment:
Probably I should remove the space from this constant and concatenate it when building the string.